### PR TITLE
Add integration tests to CI

### PR DESCRIPTION
**Pull request by jeffreyssmith2nd**
## Change Description

Depends on #965 to pass integration tests

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: .cicd/pipeline.yml [hunk 1]
```diff
@@ -292,7 +292,7 @@ steps:
     agents:
       queue: "automation-eks-eos-tester-fleet"
     timeout: ${TIMEOUT:-10}
-    skip: ${SKIP_UBUNTU_18}${SKIP_UNIT_TESTS}
+    skip: ${SKIP_UBUNTU_18}${SKIP_INTEGRATION_TESTS}
```
Comment (linhuang-blockone) [inline .cicd/pipeline.yml:295]: Why is integration test to be skipped?

**Review by linhuang-blockone — APPROVED** [review]

File: .cicd/pipeline.yml (hunk 1 above)
Comment (jeffreyssmith2nd) [reply .cicd/pipeline.yml:295]: This flag gives you the option to skip the tests by setting an ENV var on a specific build. This is really useful when debugging a different part of the CI pipeline.

Comment (linhuang-blockone) [reply .cicd/pipeline.yml:295]: Thanks for your explanation! Didn't realize it is a flag.

**Review by scottarnette — APPROVED** [review]
Noted the comment that this will fail until a different PR is merged. If that change isn't merging soon, may just want to set the SKIP flag temporarily.
🍕

**Pull request merged**